Nil pointer dereference running `velero version`

**What steps did you take and what happened:**
Run `velero version`


**What did you expect to happen:**
To show the version, or if there is a misconfiguration on my side, show an error. But don't run into a nil pointer dereference

**The output of the following commands will help us better understand what's going on**:
```
$ velero version
Client:
	Version: v1.5.1
	Git commit: -
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x247d8ba]

goroutine 1 [running]:
github.com/vmware-tanzu/velero/pkg/cmd/cli/serverstatus.(*DefaultServerStatusGetter).GetServerStatus(0xc0005181e0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/velero-20200920-59337-ij5zi3/velero-1.5.1/src/github.com/vmware-tanzu/velero/pkg/cmd/cli/serverstatus/server_status.go:44 +0x23a
github.com/vmware-tanzu/velero/pkg/cmd/cli/version.printVersion(0x2daa600, 0xc00018a008, 0x0, 0x0, 0x0, 0x2da87e0, 0xc0005181e0)
	/private/tmp/velero-20200920-59337-ij5zi3/velero-1.5.1/src/github.com/vmware-tanzu/velero/pkg/cmd/cli/version/version.go:76 +0x29c
github.com/vmware-tanzu/velero/pkg/cmd/cli/version.NewCommand.func1(0xc0004d2580, 0x3afcca0, 0x0, 0x0)
	/private/tmp/velero-20200920-59337-ij5zi3/velero-1.5.1/src/github.com/vmware-tanzu/velero/pkg/cmd/cli/version/version.go:57 +0x19b
github.com/spf13/cobra.(*Command).execute(0xc0004d2580, 0x3afcca0, 0x0, 0x0, 0xc0004d2580, 0x3afcca0)
	/private/tmp/velero-20200920-59337-ij5zi3/velero-1.5.1/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:844 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc0005b2000, 0x6, 0xc0005b2000, 0x6)
	/private/tmp/velero-20200920-59337-ij5zi3/velero-1.5.1/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:945 +0x336
github.com/spf13/cobra.(*Command).Execute(...)
	/private/tmp/velero-20200920-59337-ij5zi3/velero-1.5.1/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:885
main.main()
	/private/tmp/velero-20200920-59337-ij5zi3/velero-1.5.1/src/github.com/vmware-tanzu/velero/cmd/velero/main.go:34 +0x93
```


**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

--- a/pkg/client/factory.go
+++ b/pkg/client/factory.go
@@ -155,6 +155,10 @@ func (f *factory) KubebuilderClient() (kbclient.Client, error) {
 		Scheme: scheme,
 	})
 
+	if err != nil {
+		return nil, err
+	}
+
 	return kubebuilderClient, nil
 }
 

